Cargo.toml

```diff
@@ -12,3 +12,40 @@ exclude = [
   "wasm/nostd",
 ]
 
+[workspace.dependencies]
+generic-array = "0.14"
+
+hex = { version = "0.4", default-features = false }
+
+rand = "0.8"
+rand_core = { version = "0.6", default-features = false }
+
+serde = { version = "1", default-features = false }
+serde_json = "1"
+serde_with = { version = "2", default-features = false }
+
+sha2 = { version = "0.10", default-features = false }
+
+subtle = { version = "2.4", default-features = false }
+
+udigest = { version = "0.2.1", default-features = false }
+
+zeroize = { version = "1", default-features = false }
+
+criterion = "0.5"
+generic-tests = "0.1"
+serde_test = "1"
+rand_dev = "0.1"
+
+[workspace.dependencies.curve25519]
+package = "curve25519-dalek"
+version = "4"
```

Comment:
This line is what confuses the changelog detector. Maybe shall we write it in the `{}` style?

Reply:
It's a workspace cargo toml, it should not be looked at in the first place...

Reply:
Looking at the script, it only checks the cargo toml when there's changlelog in the same directory, so this cargo toml is not checked